backend/src/controllers/tests/getTestResults.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from 'express';
+
+import { PASS_THRESHOLD } from 'config/constants';
+import { getStatusProvider } from 'db/providers/clientCourseProvider';
+import { getTrueAnswersProvider } from 'db/providers/testProvider';
+import CourseStatus from 'enums/coursesEnums';
+import { TMiddlewareCall } from 'interfaces/commonMiddleware';
+import { isError } from 'utils/typeGuards/isError';
+import { checkTestResults, countTestResult, IAnswer } from 'utils/userTests/userTests';
+
+const getTestResults = async (
+  req: Request<Record<string, never>, Record<string, never>, { id: string; answers: IAnswer[] }>,
+  res: Response<unknown, { id: string; result: number }>,
+  next: TMiddlewareCall,
+) => {
+  try {
+    const { id: testId, answers } = req.body;
+    const { id: courseId } = req.params;
+    const courseStatus = await getStatusProvider(courseId);
+    if (!courseStatus || courseStatus?.status !== CourseStatus.testing) {
+      res.json({ message: 'test is not started' });

```

Comment:
Error maybe?

Reply:
it's not an error

Reply:
Shouldn't it be?

Reply:
Yep, but in my opinion it would be easier to implement frontend reaction on this warning if it is an error. Otherwise, they have to check the content of response, because statusCode is `OK`